### DISCUSSION TITLE
Questions about pyinstaller

### DISCUSSION THREAD

**Discussion by hoonlight** [Purfview/whisper-standalone-win#25] [Q&A] [answered]
Hi, I'm trying to package faster-whisper using pyinstaller, I'm facing a difficulty.
You seem to have done it successfully, so I'd like to ask for your advice.

If you don't mind me asking, how did you make pyinstaller find `cublas`, `cudnn`?

Just drop them in the same place as the exe file? Or do I need some other command when running pyinstaller?

I apologize for bothering you, but I'm having a hard time getting an answer after looking and looking. 
It would be even better if you could share the `.spec` file.
Thanks for a little help for a newbie, and have a nice day.

**Comment by Purfview** [comment, marked as answer]
> how did you make pyinstaller find cublas, cudnn?

I didn't. Just have those libs at your exe or in PATH.

**Reply by hoonlight** [reply]
Thanks, I'll give it a try.
